radicle-cli/src/commands/issue.rs

```diff
@@ -174,18 +174,18 @@ impl Args for Options {
         let op = match op.unwrap_or_default() {
             OperationName::Open => Operation::Open { title, description },
             OperationName::Show => Operation::Show {
-                id: id.ok_or_else(|| anyhow!("an issue id must be provided"))?,
+                id: id.ok_or_else(|| anyhow!("an issue-id must be provided"))?,
             },
             OperationName::State => Operation::State {
-                id: id.ok_or_else(|| anyhow!("an issue id must be provided"))?,
+                id: id.ok_or_else(|| anyhow!("an issue-id must be provided"))?,
                 state: state.ok_or_else(|| anyhow!("a state operation must be provided"))?,
             },
             OperationName::React => Operation::React {
-                id: id.ok_or_else(|| anyhow!("an issue id must be provided"))?,
+                id: id.ok_or_else(|| anyhow!("an issue-id must be provided"))?,
                 reaction: reaction.ok_or_else(|| anyhow!("a reaction emoji must be provided"))?,
             },
             OperationName::Delete => Operation::Delete {
-                id: id.ok_or_else(|| anyhow!("an issue id to remove must be provided"))?,
+                id: id.ok_or_else(|| anyhow!("an issue-id to remove must be provided"))?,
```

Comment:
`an issue to remove`... etc.

Reply:
https://github.com/radicle-dev/heartwood/pull/445/commits/6d0b338c6920de263d264d7bac5e5c819bc80511#diff-e5e098448f17f6e911b43e7d9c468c56af527dc6da611d21e211b94d96c21b0aR188